src/media/css/app-list-filters.styl

```diff
@@ -90,11 +90,18 @@ App listing filters.
     display: none;
 }
 
-.app-list-filters + .app-list {
+body:not([data-mkt-nav--enabled]) .app-list-filters + .app-list {
```

Comment:
Maybe we should set `data-mkt-nav="enabled"` instead? I don't like using `:not` selectors but maybe I'm a Luddite.